dspace-sword/pom.xml

```diff
@@ -121,22 +121,14 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>log4j</groupId>
-            <artifactId>log4j</artifactId>
-            <exclusions>
```

Comment:
Do we still need to have these exclusions? I am not sure where to find information about which part of log4j has these dependencies (ie api or core)